entity-management-web/src/test/java/eu/europeana/entitymanagement/web/EMControllerTest.java

```diff
@@ -52,5 +55,23 @@ public void registerEntity_concept() throws Exception {
 //	assertEquals(1, res);
 	
     }
+    
+    @Test
+    public void disableEntity() throws Exception {
+    	MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
```

Comment:
this integration test needs to ensure that the entity exists and it is not disabled at the begining of the test. This would be better achieved by creating a new entity. For this is sufficient to call the save method from the service,  for this I would suggest to create an own method createTestObject. This object must be deleted by the end of the test using the entityRecordRepo.deleteForGood()

Reply:
I have created the test in the EMControllerIT class.